app/config/urls.py

```diff
@@ -17,9 +17,9 @@
 from django.contrib import admin
 from django.urls import path
 
-from .api import api
+from .api import root
 
 urlpatterns = [
-    path('api/', api.urls),
+    path('api/', root.urls),
```

Comment:
:thinking: Not sure this is correct.

As I understand it, now we have only `/api/checker` registered. But instead we want
- Endpoint for the users: `/api/providers`, `/api/datasets` etc
- Endpoint for infra: `/checker`

So shouldn't it be something like this instead?

```py
urlpatterns = [
    path('api/', api.urls),
    path('/', root.urls),
    path('admin/', admin.site.urls),
]
```

Reply:
Thanks, messed it up. Changed it to not use NinjaAPI for the checker as discussed in person.